README.md

```diff
@@ -50,6 +55,17 @@ You will be able to see the users profile page at `http://localhost:3000/contrib
 
 3. To change the colors, fonts, or plugins edit the `tailwind.config.js` file.
 
+### Installing new packages
+1. Install `pnpm`:
+```bash
+ npm install -g pnpm
+```
+2. Install/Modify required packages
+3. Update dependencies in `pnpm-lock.yaml` file:
+```bash
+pnpm install
+```
```

Comment:
Instead of these three steps, we can just simply say: `pnpm add <package_name>` command to be ran

Reply:
but then people usually use node 
so then sometimes pnpm doesn't update 
and many systems don't have pnpm

Reply:
I will see and edit it tho

Reply:
We could have something like this maybe

<img width="938" alt="image" src="https://github.com/coronasafe/leaderboard/assets/25143503/f2b9c4fd-8f49-4828-ab5d-97511108491f">

And mention to use pnpm to add/remove packages. We don't need to mention how to install pnpm since it's already documented by pnpm and is out of the scope of the leaderboard project.

Reply:
done